flatsurf/geometry/chamanara.py

```diff
@@ -1,14 +1,16 @@
-r""" 
+r"""
 Construction of Chamanara's surfaces which depend on a parameter alpha less than one.
-See the paper "Affine automorphism groups of surfaces of infinite type" in which the surface 
+See the paper "Affine automorphism groups of surfaces of infinite type" in which the surface
 is called $X_\alpha$.
 
 EXAMPLES::
 
     sage: from flatsurf import translation_surfaces
     sage: s = translation_surfaces.chamanara(1/2)
-    sage: s.plot()     # not tested (problem with matplotlib font caches on Travis)
+    sage: s.plot()
+    ...
     Graphics object consisting of 129 graphics primitives
```

Comment:
@videlec, does this look essentially correct to you?

![image](https://user-images.githubusercontent.com/373765/202669995-a191cbb1-f8f1-4cf6-bb54-62282b19648c.png)

The old doctest had 129 primitives, now there are only 91.